sources/tools/Stride.Importer.Assimp/MeshConverter.cs

```diff
@@ -844,14 +868,17 @@ private unsafe MeshInfo ProcessMesh(Scene* scene, Silk.NET.Assimp.Mesh* mesh, Di
             var vertexBufferBinding = new VertexBufferBinding(GraphicsSerializerExtensions.ToSerializableVersion(new BufferData(BufferFlags.VertexBuffer, vertexBuffer)), vertexDeclaration, (int)mesh->MNumVertices, vertexDeclaration.VertexStride, 0);
             var indexBufferBinding = new IndexBufferBinding(GraphicsSerializerExtensions.ToSerializableVersion(new BufferData(BufferFlags.IndexBuffer, indexBuffer)), is32BitIndex, (int)nbIndices, 0);
 
-            var drawData = new MeshDraw
-            {
-                VertexBuffers = new VertexBufferBinding[] { vertexBufferBinding },
-                IndexBuffer = indexBufferBinding,
-                PrimitiveType = PrimitiveType.TriangleList,
-                DrawCount = (int)nbIndices
-            };
-
+           
+            //drawData.VertexBuffers =;
+            //{
+                drawData.VertexBuffers = new VertexBufferBinding[] { vertexBufferBinding };
+                drawData.IndexBuffer = indexBufferBinding;
+                drawData.PrimitiveType = PrimitiveType.TriangleList;
+                drawData.DrawCount = (int)nbIndices;
+            //}
+
```

Comment:
Remove comments and fix indentation

Reply:
This project should be removed in latest .sln file